corehq/apps/userreports/indicators/specs.py

```diff
@@ -87,10 +90,3 @@ class ChoiceListIndicatorSpec(PropertyReferenceIndicatorSpecBase):
 
     def get_operator(self):
         return IN_MULTISELECT if self.select_style == 'multiple' else EQUAL
-
-
-def _transform_from_datatype(datatype):
-    return {
-        'date': transform_date,
-        'integer': transform_int,
-    }.get(datatype)
```

Comment:
In the short term we could make the default transform "identity" which just returns `lambda x: x`